### DISCUSSION TITLE
Dynamic scaling render resolution as of today?

### DISCUSSION THREAD

**Discussion by Bketting** [google/model-viewer#2109] [Q&A]
Were there changes made to how models are rendered today? As of today, models seem to scale dynamically with lower resolutions when zoomed further out from the object. This wasn't the case when yesterday (or even this morning CET). See below for an example (iPhone 12):

**Smooth(er) line when zoomed-in**
![Close-up](https://i.postimg.cc/BvcD8Fx6/IMG-3764.png)
**Distorted line when zoomed-out**
![Zoom-out](https://i.postimg.cc/nzFmqLk1/IMG-3765.png)

**Comment by elalish** [comment]
Those screenshots both look pretty good to me; can you maybe circle exactly where you think there's a problem? And as long as you're using unpkg to get model-viewer releases, we haven't made one in a month. Keep in mind that Safari could easily have changed something too; they have numerous bugs related to WebGL.

**Reply by Bketting** [reply]
The problem was the edgy / distorted shape of the line within the pink plastic part on the bottom photo. About 1 hour after posting this message everything was back to normal though. So likely something at the Safari-side like you mentioned. Thank you
